File: src/main/java/com/tom/Hello.java
package com.tom;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello word");
        Person p = new Person("Ryuu", 66.5f,1.7f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80 || score >90);
        char c ='A';
        System.out.println(c> 'a');
       /* int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        double height = 1.7;
        boolean adult = true;
        boolean enroll = false;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main(args: Array<String>) {
//    println("Hello kotlin")
    val h = Human( 66.5f,1.7f)
    println(h.bmi())
    /*var age = 19;
    age = 20;
    var weight = 66.5f;
    var name: String;
    name = "Hank"*/
}

class Human(var weight: Float, var height:Float) {

    fun bmi() :Float{
        val bmi = weight / (height*height)
        return  bmi
    }
    fun hello() {
        println("Hello kotlin")
    }
}